api/src/controllers/controllerUtil.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { NextFunction, Request, Response } from 'express';
+
+type Handler = (req: Request<any>, res: Response<any>, next: NextFunction) => Promise<void>;
```

Comment:
I'll spend more time testing this on the weekend but I'm pretty sure I do b/c I remember trying without `any` and had no luck.

They need to be generically typed because this helper will be called on controllers with all sorts of different generic types for their requests and responses. It can't be `uknown` because `unknown` can have anything assigned to it but can't assign to anything.

Reply:
Hey turns out you were write. I could just entirely remove the generic part of the type and then I didn't need `any`.